### ISSUE TITLE
"Reorder Categories" dialog slightly offscreen

### ISSUE THREAD

**Issue by amotl** [Daemonite/discourse-material-theme#8]
## Problem
The "Reorder Categories" dialog seems to be rendered slightly offscreen, expanding beyond the viewport towards the top.

However, editing and saving works, so this is just a minor cosmetic thing.

Hope this helps.

**Comment by amotl** [comment]
## Workaround
While this was declared already, it got overridden by some `max-height: none !important` rule coming from Discourse. This patch adds the original height computation again, which is obviously working correctly as intended.
```css
/* Bugfix: "Reorder Categories" dialog slightly off */
.modal .modal-body.full-height-modal.reorder-categories {
    max-height: calc(100vh - 150px) !important;
}
```

So, there's not really something to fix here but to investigate why this rule gets overwritten by a nonsense one.

**Comment by sesemaya** [comment]
This should have been fixed by 4cbc1d3.
